src/components/donationItemForm/index.tsx

```diff
@@ -81,9 +74,24 @@ export default function DonationItemForm({
     }
   });
 
+  // Checking undefined, null, and emptiness before show the Higg/Low values
+  const updateHighLowVals = (
+    newOrUsed: string,
+    value: undefined | ItemResponse
+  ) => {
+    if (newOrUsed === 'Used' && typeof value === 'undefined') {
+      setHighLowVals(['High', 'Low']);
+    } else if (newOrUsed === 'Used' && typeof value !== 'undefined') {
+      setHighLowVals([`High ($${value?.high})`, `Low ($${value?.low})`]);
+    }
+  };
+
```

Comment:
Would be a bit cleaner if `value` was an optional argument. ex

`const updateHighLowVals = (newOrUsed: string, value?: ItemResponse) => ...`

Then you can check if it is present with `if ([condition 1] && value)`

That way we avoid any sentinel values, and we aren't passing `undefined` into functions